src/AppStakers.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect } from "react"
+import { Outlet } from "react-router-dom"
+import { useAccount } from "wagmi"
+import { Footer } from "./components/Footer"
+import { Header, NavigationLink } from "./components/Header"
+import { useFundraisePosition } from "./hooks/api/useFundraisePosition"
+import { routes } from "./utils/routes"
+
+import styles from "./App.module.scss"
+
+const AppStakers = () => {
+  const [{ data: accountData }] = useAccount()
+  const { getFundraisePosition, data: fundraisePositionData } = useFundraisePosition()
+
+  useEffect(() => {
+    if (accountData?.address) {
+      getFundraisePosition(accountData.address)
+    }
+  }, [accountData?.address, getFundraisePosition])
+
+  const navigationLinks: NavigationLink[] = [
+    {
+      title: "OVERVIEW",
+      route: routes.Overview,
+    },
+    {
+      title: "STAKE",
+      route: routes.Stake,
+    },
+    {
+      title: "POSITIONS",
+      route: routes.Positions,
+    },
+    {
+      title: "PROTOCOLS",
+      route: routes.Protocols,
+      external: true,
+    },
+  ]
+
+  if (fundraisePositionData) {
+    navigationLinks.push({
```

Comment:
I think it would look better with the CLAIM position before the PROTOCOLS; what do you think?

Reply:
100%. Good catch!